src/main/modules/appinsights/index.ts

```diff
@@ -3,16 +3,12 @@ import config from 'config';
 const appInsights = require('applicationinsights');
 
 export class AppInsights {
-
   enable(): void {
     if (config.get('appInsights.instrumentationKey')) {
-      appInsights.setup(config.get('appInsights.instrumentationKey'))
-        .setSendLiveMetrics(true)
-        .start();
+      appInsights.setup(config.get('appInsights.instrumentationKey')).setSendLiveMetrics(true).start();
```

Comment:
The Prettier recommended default for `printWidth` is `80`.

This can be configured to be [shorter](https://prettier.io/docs/en/options.html#print-width), which would wrap these lines, but then would also wrap other parts of the code.